app/Http/Controllers/CompanyController.php

```diff
@@ -22,51 +22,33 @@ public function draw(Request $request){
         $start = $request->query('start', 0);
         $length = $request->query('length', 10);
         $totalCompanys =   Company::count();
-        $dTcolumns = $request->query('columns');;
-        $filterDropDownValues = array();
-
-        if (is_null($search)) {
-            for ($x = 0; $x <= 6; $x++) {
-                if (!is_null($dTcolumns[$x]['search']['value'])) {
-                    $filterDropDownValues[$dTcolumns[$x]['data']] = $dTcolumns[$x]['search']['value'];
-                };
-            }
-        }
-
-        if(!empty($filterDropDownValues)){
-            $searchCountry = array_key_exists('country', $filterDropDownValues) ? $filterDropDownValues['country'] : null;
-            $companys = Company::where('country', 'like', "%" . $searchCountry . "%");
 
-        }else{
+        if(!$request->dropdowns){
             $companys = Company::where('name', 'like', "%" . $search . "%")
                 ->orWhere('country', 'like', "%" . $search . "%")
                 ->orWhere('branch', 'like', "%" . $search . "%")
                 ->orWhere('address', 'like', "%" . $search . "%");
+        }else{
+            $searchCountry = $request->dropdowns['country'] ?? null;
```

Comment:
use equal conditions for filter dropdown

Reply:
Sure did @Raza9798 please review again

Reply:
Which is better approach?
![image](https://github.com/user-attachments/assets/c894eb7a-e422-4d15-a0d6-568d6ba80165)

![image](https://github.com/user-attachments/assets/e8958b10-cf85-495e-8921-45218780cb89)

Reply:
this condition won't work if we choose multiple items in the filter please fix it

Reply:
Fixed please review again @Raza9798